launch_testing/launch_testing/__init__.py

```diff
@@ -15,25 +17,31 @@ class InMemoryHandler(LineOutput):
         that will only need one line to match, instead of the entire output.
     @param regex_match: If true, treat the expected_lines as a regular expression in match
         accordingly.
-    @param filtered_prefixes: A list of prefixes that will be ignored. By default the output of
-        RTI Connext will be ignored.
+    @param filtered_prefixes: A list of prefixes that will cause output lines to be ignored if 
+        they start with one of the prefixes. By default lines starting with 
+        the process ID (`pid`) and return code (`rc`) will be ignored.
+    @param rmw_implementations_to_filter: A list of RMW implementations for which the output will
+        be ignored in addition to the default/``filtered_prefixes``.
     @param exit_on_match: If True, then when its output is matched, this handler
         will terminate; otherwise it will simply keep track of the match.
+    :raises: :exc:`LookupError`
```

Comment:
is it ok to mix the @ and : styles here? should I change all lines to use : as is done [here](https://github.com/ament/ament_index/blob/master/ament_index_python/ament_index_python/__init__.py#L46)?